cmd/helm/list.go

```diff
@@ -150,19 +165,20 @@ func (l *listCmd) run() error {
 		return nil
 	}
 
-	if res.Next != "" && !l.short {
+	if res.Next != "" && !l.short && l.format == "text" {
 		fmt.Fprintf(l.out, "\tnext: %s\n", res.Next)
 	}
 
-	rels := res.Releases
-
+	format := l.format
 	if l.short {
-		for _, r := range rels {
-			fmt.Fprintln(l.out, r.Name)
-		}
-		return nil
+		format = "short"
+	}
+
+	formatted, err := formatList(res.Releases, format)
+	if err != nil {
+		return err
 	}
-	fmt.Fprintln(l.out, formatList(rels))
+	fmt.Fprint(l.out, formatted)
```

Comment:
have you tested this change with the text and short output formats? Looks fine, but just want to make sure that this has been tested.